### DISCUSSION TITLE
提前 5~25 秒示警，AI 地震預警系統晶圓廠也買單

### DISCUSSION THREAD

**Discussion by Yao900108** [1101-GS4719-C/Python-Community-GS4719-C#561] [科技新知]
科技部 20 日展現 2 組學研新創成果，衛波科技研發「AI 現地型地震預警系統」可在地震前 5~25秒示警，降低精密設備損壞，產線復歸可望從數小時縮短到15~30分鐘。
過去國研院地震中心的研究員不斷鑽研地震相關研究，在科技部科研創業計畫輔導下，成立衛波科技，研發出業界首創的「AI現地型地震預警系統」與「結構監測自動化平台」，希望做到震前預警、震後監測、加速救災效率等。
地震預警的原理是先偵測速度較快抵達的P波，再來預測後續地震的大小。「AI現地型地震預警系統」可以提前5~25秒預警地震，同時完成自動化控制，降低廠商精密設備損害，目前台灣高鐵跟多家台灣高科技大廠採用，包含晶圓大廠、光罩廠還有顯示玻璃廠等

https://technews.tw/2022/04/21/ai-earthquake-warning-system/

**Comment by ghost** [comment]
看到P波就想到已經被忘掉的地科....
不過偵測地震只能說有其極限，因為他是發生後才發出波，但反過來說，能有多一秒的準備就有多一分的保險

**Comment by Ku-Hao-Chen** [comment]
謝謝分享，希望可以真正實用在我們的生活。

**Comment by sean1325418** [comment]
其實就是看是電子訊號傳得快還是地震波傳得快而已

**Comment by Sazmafuji** [comment]
地震中的一秒就很緊要，希望此5-25秒能夠救許多人

**Comment by PoooYen** [comment]
也要配合細胞簡訊的快速擴散才可以有效的救更多人

**Comment by yi-1111** [comment]
謝謝分享，希望這5-25秒的預警，減少社會上損失

**Comment by HuanYuuuuuu** [comment]
在地震時能提早1秒有所準備就可以拯救許多人了!

**Comment by Jay-722** [comment]
花蓮人的話，提前告訴他，他也不會跑

**Comment by Ben6uo** [comment]
謝謝分享，地震真的好可怕，之前全臺地震真的嚇死我了

**Comment by Yuhsin000** [comment]
地震震一下，股價也震一下，要好好保護器材QQ

**Comment by Lai27330** [comment]
謝謝分享，多了這幾秒可以拯救到更多的人

**Comment by hapiwangy** [comment]
台灣處於環太平洋火山地震帶上，相信這類型的系統可以幫忙台灣許多

**Comment by HsuChen88** [comment]
之前收到通知覺得5秒也來不及跑，原來這時間是要給精密機器拿來緊急關閉的。

**Comment by handsomeniko** [comment]
話說 現在最新科技是用重力波觀測地震 
就能用光速傳遞了

**Comment by 110409538** [comment]
提前預警就可以增加大家安全撤離的時間。

**Comment by jcpan821** [comment]
若是真的能提前25秒預警，那真的能救到很多人，但像前幾年那樣都在晚上發生，如果不是社區型廣播警告，而是手機的震動的話，可能還是難以察覺吧

**Comment by d26488399** [comment]
多了幾秒的時間可以讓人有時間做出應對措施

**Comment by yucheng1028** [comment]
感覺這蠻重要的

**Comment by cordydeng** [comment]
提前5~25秒的確是能夠救很多人的黃金時間，對於高精密儀器來說，能盡量減少地震造成的損害也意味著能省下一筆費用

**Comment by henrywen547** [comment]
提前預警可以增加不少撤離的時間，對於精密設備的維護沒想到能節省這麼多時間!
話說兩個~要怎麼判斷是或不是刪除線啊

**Comment by Rosancl** [comment]
面對地震，每能提前多一秒預測就是好事~~~

**Comment by yuhao112** [comment]
在台灣確實是很需要這個技術呢

**Comment by SamTU0222** [comment]
謝謝分享，有了這短短的時間可以人們提早去規劃要如何去應對不同大小的地震漸少損失

**Comment by stan890425** [comment]
晶圓廠只要地震就損失慘重，這的確對他們來說是必須的

**Comment by leon10456** [comment]
感謝樓主的分享

**Comment by HaoFish123** [comment]
感謝分享，但我想問題應該不是能提前多少秒，而是要怎麼有效快速的通知。

**Comment by treeann** [comment]
AI真的太多用途了...

**Comment by alexthebigpp** [comment]
提前5到25秒感覺就可以做很多事情了。

**Comment by yenwei12138** [comment]
5~25秒真的很多，地震預測的技術台灣已發展多年，果然是越來越有成效了

**Comment by etin2468** [comment]
這對這種產值很高的廠房相當重要阿

**Comment by bloodtrail0511** [comment]
感謝分享

**Comment by Lynn12378** [comment]
訊息的傳遞速度進步確實帶給人們很大的福祉
